client/src/components/LookupField/tests/LookupField-test.js

```diff
@@ -1,12 +1,11 @@
 /* global jest, describe, beforeEach, it, expect, require */
 
-jest.unmock('react');
-jest.unmock('react-dom/test-utils');
-jest.unmock('../LookupField');
-
 import React from 'react';
-import ReactTestUtils from 'react-dom/test-utils';
```

Comment:
ReactTestUtils got bumped to a newer version that fires off an error because of some rogue input statement in reactstrap. My guess is that we can't fix this issue without upgrading reactstrap. So I'm switching this test to enzyme instead.